tests/kafkatest/services/kafka/templates/kafka.properties

```diff
@@ -30,7 +30,6 @@ num.partitions=1
 num.recovery.threads.per.data.dir=1
 log.retention.hours=168
 log.segment.bytes=1073741824
-log.retention.check.interval.ms=300000
```

Comment:
Why remove this? I'd have though that we'd make it configurable if anything.

Reply:
Removing this makes it configurable (I override it in my test). Otherwise, these values are appended at the end of the properties file, which means they can't be changed in the tests.